### DISCUSSION TITLE
Can I use github pages as a virtual youtuber site?

### DISCUSSION THREAD

**Discussion by NyaTube** [community/community#21963] [Other Feature Feedback, Questions, & Ideas]
<p>I want to publish my virtual youtuber website on github pages, but I want to know if it violates the online business prohibition.<br>
There is only a description of the virtual youtuber and a link to YouTube, but no goods or other items for sale.</p>
<p>*im not a virtual youtuber.</p>

**Comment by jfullstackdev** [comment, marked as answer]
<blockquote>
<h3>Prohibited uses</h3>
<p>GitHub Pages is not intended for or allowed to be used as a free web-hosting service to run your online business, e-commerce site, or any other website that is primarily directed at either facilitating commercial transactions or providing commercial software as a service (SaaS). GitHub Pages sites shouldn’t be used for sensitive transactions like sending passwords or credit card numbers.</p>
<p>In addition, your use of GitHub Pages is subject to the <a href="https://docs.github.com/en/free-pro-team@latest/github/site-policy/github-terms-of-service">GitHub Terms of Service</a>, including the restrictions on get-rich-quick schemes, sexually obscene content, and violent or threatening content or activity.</p>
</blockquote>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages">
  <header class="source">
      <img src="https://docs.github.com/assets/cb-803/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages" target="_blank" rel="noopener">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/47092464/181067919-30f0456c-7865-45b6-a2ee-1a783549837d.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages" target="_blank" rel="noopener">About GitHub Pages - GitHub Docs</a></h3>

  <p>You can use GitHub Pages to host a website about yourself, your organization, or your project directly from a repository on GitHub.com.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>in my opinion, you can use GitHub to ask for donations but not full business, simple ads too will do, because GitHub Sponsor is functioning the same thing,</p>
<p>the full business, is say, running a website that is all about business products</p>

**Comment by NyaTube** [comment]
<p>I see, thanks for the help.</p>
